box-content-sample/src/main/java/com/box/androidsdk/sample/MainActivity.java

```diff
@@ -68,8 +68,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private void initialize() {
         mAdapter.clear();
-
-        mSession = new BoxSession(this, null);
+        mSession = new BoxSession(this);
```

Comment:
This is a different behavior than before as this version always logs in to the last used account.